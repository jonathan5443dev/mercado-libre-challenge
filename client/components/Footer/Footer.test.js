import React from "react";
import Footer from "./Footer";

import renderer from "react-test-renderer";
import LightTheme from "@config/themes/light";
import { ThemeProvider } from "styled-components";

describe("Footer", () => {
  test("should render", () => {
    const component = renderer.create(
      <ThemeProvider theme={LightTheme}>
        <Footer copy="Texto de prueba" onClick={() => console.log("cambio")} />
      </ThemeProvider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
