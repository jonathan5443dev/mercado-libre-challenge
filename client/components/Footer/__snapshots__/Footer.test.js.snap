// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<div
  className="g8qojs-0 kQTLEp"
>
  <div
    className="sc-1os0qt1-0 ixyyBw"
    direction="row"
  >
    <p
      className="sc-1hvrgbn-6 eYrVMp"
      color="normal"
      fontWeight="normal"
      type="small"
    >
      Texto de prueba /
    </p>
    <div
      className="g8qojs-1 dHIyst"
      onClick={[Function]}
    >
      <p
        className="sc-1hvrgbn-6 eYrVMp"
        color="normal"
        fontWeight="normal"
        type="small"
      >
        Cambiar tema
      </p>
    </div>
  </div>
</div>
`;
